### PR TITLE
f-checkout@0.5.0 - Replicate Collection Layout

### PR DESCRIPTION
**Pull request by FayeCarter**
v0.5.0
------------------------------
*October 21, 2020*

### Added
- Collection component to replicate ConsumerWeb.
- Delivery component to replicate ConsumerWeb.

## UI Review Checks

image

- [ ] README and/or UI Documentation has been [created|updated]
- [x] Unit tests have been [created|updated]
- [ ] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

### CODE REVIEW

File: packages/f-checkout/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.5.0
+------------------------------
+*October 21, 2020*
+
+### Added
+- Collection component to replicate ConsumerWeb.
+- Delivery component to replicate ConsumerWeb.
```
Comment (SachaZvetelman) [inline packages/f-checkout/CHANGELOG.md:12]: Wasn't that already in the previous version? 😬

File: packages/f-checkout/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-checkout",
   "description": "Fozzie Checkout – Fozzie Checkout Component",
-  "version": "0.2.0",
+  "version": "0.5.0",
```
Comment (SachaZvetelman) [inline packages/f-checkout/package.json:4]: Why are we jumping from `0.2.0` to `0.5.0`, or are there other PRs in between?

File: packages/f-checkout/src/components/Delivery.vue [hunk 3]
```diff
@@ -0,0 +1,99 @@
+<template>
+    <div data-test-id='delivery-component'>
+        <form-field
+            v-model="mobileNumber"
+            :class="$style['c-formField']"
+            name="mobile-number"
+            data-test-id="input-mobile-number"
+            :label-text="labels.mobileNumber"
+            input-type="text"
+            label-style="inline" />
```
Comment (SachaZvetelman) [inline packages/f-checkout/src/components/Delivery.vue:10]: This is shared between delivery and collection. Maybe we can extract this to `Checkout.vue`?

File: packages/f-checkout/CHANGELOG.md (hunk 1 above)
Comment (SachaZvetelman) [reply packages/f-checkout/CHANGELOG.md:12]: Ah, ignore me.

File: packages/f-checkout/package.json (hunk 2 above)
Comment (FayeCarter) [reply packages/f-checkout/package.json:4]: There's two in the middle. One that's just been merged and Ben is writing some component tests

**Pull request closed without merging**